doc/changes/latest.inc

```diff
@@ -116,6 +116,8 @@ Bugs
 
 - When creating `~mne.Epochs`, we now ensure that ``reject_tmin`` and ``reject_tmax`` cannot fall outside of the epochs' time interval anymore (:gh:`8821` by `Richard Höchenberger`_)
 
+- :meth:`raw.rename_channels() <mne.io.Raw.rename_channels>` now also applies the new channel names to ``raw._orig_units``, if present (:gh:`8846` by `Richard Höchenberger`_)
```

Comment:
I would not reference private properties in changelogs, users shouldn't use or care about them, and mentioning them makes it seem like maybe they should or maybe could even make use of them somehow. Instead it's better to say that you've fixed a bug in accounting for the original units or whatever the downstream, public-API or behavior result is